src/benchmarks/micro/corefx/System.Memory/ReadOnlySpan.cs

```diff
@@ -73,7 +73,18 @@ public int IndexOfString(string input, string value, StringComparison comparison
             new object[] { GenerateInputString('\u3060', 1000, 'x', 500), "x", StringComparison.Ordinal },
             new object[] { GenerateInputString('\u3060', 100, '\u3059', 50), "\u3059", StringComparison.Ordinal }
         };
-        
+
+        [Benchmark]
+        [ArgumentsSource(nameof(TrimArguments))]
+        public ReadOnlySpan<char> Trim(string input) => input.AsSpan().Trim();
+
+        public static IEnumerable<object[]> TrimArguments => new List<object[]>
```

Comment:
I was just coming to write this comment. :)

Reply:
> If the method accepts a single argument, the ArgumentsSource should return IEnumerable<object> instead of IEnumerable<object[]> (not an array).

What about the IndexOfStringArguments above this?  I just copied it from there.

Reply:
hehe. BTW @billwert sorry for the delay in reviews this week, I got totally sucked by the Team Week activities.